lib/_stream_writable.js

```diff
@@ -339,6 +337,8 @@ function onwrite(stream, er) {
     }
 
     if (sync) {
+      // TODO(trevnorris): Yank this, but first need to figure out why
```

Comment:
I just tested this and all tests still pass after changing this additional instance of `process.nextTick()` to `process.nextTick(afterWrite, stream, state, finished, cb);`